drivers/openstack/openstack.go

```diff
@@ -318,6 +319,35 @@ func (d *Driver) GetIP() (string, error) {
 	return "", fmt.Errorf("No IP found for the machine")
 }
 
+func (d *Driver) GetPrivateIP() (string, error) {
```

Comment:
Did you test this in conjunction with a floating IP of `assignFloatingIP()`?

How does the command look like?